subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -1287,6 +1302,19 @@ static int isr_rx_scan_report(struct lll_scan *lll, uint8_t rssi_ready,
 				ftr->radio_end_us =
 					radio_tmr_end_get() -
 					radio_rx_chain_delay_get(lll->phy, 1);
+
+				/* FIXME figure out why ext-to-aux scheduling
+				 *       from LLL still fails sometimes...
+				 */
```

Comment:
This seemed to work initially but then I found it still can fail so just commented out for now so it can be tested...